modules/apps/headless/headless-delivery/headless-delivery-test/src/testIntegration/resources/com/liferay/headless/delivery/resource/v1_0/test/dependencies/test-not-localized-ddm-structure.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
@gergelyszalay, I would use the same naming that you used in the tests, maybe test-unlocalized-ddm-structure.json